test/performance/README.md

```diff
@@ -21,3 +23,36 @@ Their name reference the test where they come from, currently `get-*.csv` and `c
 
 -   `.txt` files contain summaries of Locust, the time unit is milliseconds.
 -   `.csv` files contain values from Prometheus, the CPU unit is in milliCPU, the memory unit is bytes.
+
+## Merging results from GitHub Workflows
+
+This process merge test results form GitHub Workflows from the last 90 days so each
```

Comment:
Hey @rh-hemartin ! I'm re-reading this and I think there is a 2 typos:
"This process merges test results from GitHub Workflows ..." does this make sense?